test/support/conn_case.ex

```diff
@@ -11,18 +11,18 @@ defmodule TestDispatchForm.ConnCase do
 
   using do
     quote do
-      Application.put_env(:test_dispatch_form, TestDispatchFormTest.Endpoint, [])
+      Application.put_env(:test_dispatch_form, TestDispatchTest.Endpoint, [])
```

Comment:
```suggestion
      Application.put_env(:test_dispatch, TestDispatchTest.Endpoint, [])
```

Reply:
Nice, changed all the test_dispatch_form's